Src/WitsmlExplorer.Frontend/components/Modals/DeleteEmptyMnemonicsModal.tsx

```diff
@@ -0,0 +1,84 @@
+import Wellbore from "../../models/wellbore";
+import Well from "../../models/well";
+import JobService, { JobType } from "../../services/jobService";
+import { TextField } from "@material-ui/core";
+import OperationType from "../../contexts/operationType";
+import { useContext, useState } from "react";
+import { HideModalAction } from "../../contexts/operationStateReducer";
+import { DeleteEmptyMnemonicsJob } from "../../models/jobs/deleteEmptyMnemonicsJob";
+import ModalDialog from "./ModalDialog";
+import OperationContext from "../../contexts/operationContext";
+import { DateTimeField } from "./DateTimeField";
+import styled from "styled-components";
+
+export interface DeleteEmptyMnemonicsModalProps {
+  wells?: Well[];
+  wellbores?: Wellbore[];
+  dispatchOperation: (action: HideModalAction) => void;
+}
+
+const DeleteEmptyMnemonicsModal = (props: DeleteEmptyMnemonicsModalProps): React.ReactElement => {
+  const { wells, wellbores, dispatchOperation } = props;
+  const {
+    operationState: { timeZone }
+  } = useContext(OperationContext);
+  const [nullDepthValue, setNullDepthValue] = useState<number>(-999.25);
+  const [nullTimeValue, setNullTimeValue] = useState<string>("1900-01-01T00:00:00.000+00:00");
+  const [dTimeKickoffValid, setDTimeKickoffValid] = useState<boolean>(true);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const onSubmit = async (nullDepthValue: number, nullTimeValue: string) => {
+    setIsLoading(true);
+
+    const job: DeleteEmptyMnemonicsJob = {
+      wells: wells?.map((x) => {
+        return { wellUid: x.uid, wellName: x.name };
+      }),
+      wellbores: wellbores?.map((x) => {
+        return { wellboreUid: x.uid, wellboreName: x.name, wellUid: x.wellUid, wellName: x.wellName };
+      }),
+      nullDepthValue: nullDepthValue,
+      nullTimeValue: nullTimeValue
+    };
+
+    await JobService.orderJob(JobType.DeleteEmptyMnemonics, job);
+
+    setIsLoading(false);
+
+    dispatchOperation({ type: OperationType.HideModal });
+  };
+
+  return (
+    <>
+      <ModalDialog
+        heading={`Delete empty mnemonics`}
+        confirmText={`Submit`}
+        content={
+          <ContentLayout>
+            <DateTimeField
+              value={nullTimeValue}
+              label="Null time value"
+              updateObject={(dateTime: string, valid: boolean) => {
+                setNullTimeValue(nullTimeValue);
```

Comment:
`setNullTimeValue(nullTimeValue)` sets the nullTimeValue to the default. Should be `setNullTimeValue(dateTime)`.